components/system-broker/pkg/ord/ord_test.go

```diff
@@ -0,0 +1,51 @@
+package ord_test
+
```

Comment:
Maybe _config_test.go_ is better name for this file?

Reply:
Addressed it! 🚀